upstream-community-operators/vault/vaultoperator.v0.4.10.clusterserviceversion.yaml

```diff
@@ -0,0 +1,335 @@
+    mediatype: image/svg+xml
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+      - serviceAccountName: vault-operator
+        rules:
+        - apiGroups:
+          - vault.banzaicloud.com
+          resources:
+          - vaults
+          - vaults/finalizers
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - "*"
+        - apiGroups:
+            - ""
+          resources:
+            - namespaces
+          verbs:
+            - get
+        - apiGroups:
+            - extensions
+          resources:
+            - ingresses
+          verbs:
+            - get
+            - create
+            - update
+        - apiGroups:
+            - apps
+          resources:
+            - deployments
+            - statefulsets
+          verbs:
+            - get
+            - create
+            - update
```

Comment:
```suggestion
            - update
            - list
```

Reply:
To be more specific, the example CR tries to create a `secret` in the namespace `default` which it did not get permissions to do so.